docs/pages/modal.js

```diff
@@ -1,217 +1,141 @@
 // @flow strict
 import type { Node } from 'react';
-import PropTable from '../components/PropTable.js';
+import docgen from '../components/docgen.js';
 import PageHeader from '../components/PageHeader.js';
 import MainSection from '../components/MainSection.js';
-import CardPage from '../components/CardPage.js';
+import Page from '../components/Page.js';
+import GeneratedPropTable from '../components/GeneratedPropTable.js';
+import type { DocGen } from '../components/docgen.js';
 
-const cards: Array<Node> = [];
-const card = (c) => cards.push(c);
-
-card(
-  <PageHeader
-    name="Modal"
-    description="Modals display content that requires user interaction. They appear on a layer above the page and therefore block the content underneath, preventing users from interacting with anything else besides the Modal.
+export default function ModalPage({ generatedDocGen }: {| generatedDocGen: DocGen |}): Node {
+  return (
+    <Page title="Modal">
+      <PageHeader
```

Comment:
`<Page />` now takes regular React children, so we no longer need the `cards` prop